Doc/library/asyncio-graph.rst

```diff
@@ -0,0 +1,145 @@
+.. currentmodule:: asyncio
+
+
+.. _asyncio-graph:
+
+========================
+Call Graph Introspection
+========================
+
+**Source code:** :source:`Lib/asyncio/graph.py`
+
+-------------------------------------
+
+asyncio has powerful runtime call graph introspection utilities
+to trace the entire call graph of a running *coroutine* or *task*, or
+a suspended *future*.  These utilities and the underlying machinery
+can be used by users in their Python code or by external profilers
+and debuggers.
+
+.. versionadded:: next
+
+
+.. function:: print_call_graph(future=None, /, *, file=None, depth=1, limit=None)
+
+   Print the async call graph for the current task or the provided
+   :class:`Task` or :class:`Future`.
+
+   This function prints entries starting from the currently executing frame,
+   i.e. the top frame, and going down towards the invocation point.
```

Comment:
If `future` is not `None`, then the top frame may be suspended (so it's not the `currently executing` frame). Right?

Reply:
Yeah, correct, this is meant to say "currently executing frame from the point where the call stack capturing API was called".

Reply:
This is meant to say the top frame of either `future` or the current running task. The point I was making when writing this was that the order is reversed from what you'd typically expect in Python tracebacks.